tsconfig.production.json

```diff
@@ -14,11 +15,12 @@
     "noUnusedParameters": true,
     "outDir": "lib",
     "pretty": true,
+    "resolveJsonModule": true,
```

Comment:
Why?

Reply:
This is needed for reading the version of `package,json`, see: https://github.com/ridedott/eslint-plugin/blob/0d1b41b2e6fa7c18ddca54619c872d55e5350a02/src/util.ts#L11-L19